tests/jax/common.libsonnet

```diff
@@ -197,4 +197,32 @@ local tpus = import 'templates/tpus.libsonnet';
       |||,
     },
   },
+  huggingFaceDiffuser:: {
```

Comment:
Thanks for adding this! Do you mind updating `huggingFace` in `huggingFaceTransformers` to distinguish `huggingFaceDiffuser`?